### ISSUE TITLE
Errors are not getting reported back to the user

### ISSUE THREAD

**Issue by jan-schreib** [Qabel/qabel-desktop#441]
Problem: 
The input fields are getting validated when a new account is created, but invalid inputs are not getting reported back to user. Also happens when trying to login.

To reproduce:
1. start the programm
2. click on "Qabel account erstellen" or something like "Create Qabel account"
3. fill the fields with the value "asd" or "a".
4. press "erstellen" or "create"

**Comment by julianseeger** [comment]
To clarify: it gets reported ... but as a tooltip on the red button. Obviously not obvious enough

**Comment by enkore** [comment]
There is at least another instance of this in the Sync functionality. When an upload fails this is only visible in the window opened by the question mark button, while the fully qabel-orangized progress bar says "Ready".

**Comment by thechauffeur** [comment]
Yep enhancing the way errors are shown to the user is a very good idea :)
